src/AppInstallerCommonCore/PathVariable.cpp

```diff
@@ -121,6 +121,7 @@ namespace AppInstaller::Registry::Environment
 
         std::wstring pathName = std::wstring{ s_PathName };
         m_key.SetValue(pathName, ConvertToUTF16(value), REG_EXPAND_SZ);
+        SendNotifyMessage(HWND_BROADCAST, WM_SETTINGCHANGE, 0, (LPARAM)TEXT("Environment"));

```

Comment:
```suggestion
        SendNotifyMessageW(HWND_BROADCAST, WM_SETTINGCHANGE, 0, (LPARAM)TEXT("Environment"));

```
nit: We prefer explicitly call wide version of win32 api

Reply:
I see; I thought this was covered since the define in the remaps it to the wide version
![image](https://github.com/microsoft/winget-cli/assets/12611259/4076b148-3261-4e36-b17f-72957bd58ff4)